src/main/java/com/fundation/logic/view/resultTable/DetailsFrame.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view.resultTable;
+
+import javax.swing.JFrame;
+import java.awt.*;
```

Comment:
Not import complet packages, only needed classes.

Reply:
done